.github/workflows/codespell.yml

```diff
@@ -0,0 +1,11 @@
+# https://pypi.org/project/codespell will fix common misspellings in text files. It’s designed primarily
+# for checking misspelled words in source code, but it can be used with other files as well.
```

Comment:
```suggestion
# https://pypi.org/project/codespell will fix common misspellings in text files.
# It’s designed primarily for checking misspelled words in source code, but
# it can be used with other files as well.
```